datalad/distributed/create_sibling_ria.py

```diff
@@ -635,7 +643,12 @@ def _create_sibling_ria(
             disabled_hook.rename(enabled_hook)
         if group:
             # TODO; do we need a cwd here?
-            subprocess.run(chgrp_cmd, cwd=quote_cmdlinearg(ds.path))
+            subprocess.run(chgrp_cmd, cwd=ds.path)
+        # finally update server
+        if post_update_hook:
+            # Conditional on post_update_hook, since one w/o the other doesn't
+            # seem to make much sense.
+            gr.call_git("update-server-info")
```

Comment:
Oh no, my parenthetical, reading-along thought is causing more trouble than it was worth :)

`call_git`s first argument should be a list.

Reply:
God dammit. Don't code when ou're half way out to vacation. :-D